fiat/coincap_api.go

```diff
@@ -37,94 +30,94 @@ var (
 
 // Granularity indicates the level of aggregation price information will be
 // provided at.
-type Granularity string
+type Granularity time.Duration
```

Comment:
Is there a need to granularity at all if we standardize on 1 hour?

Reply:
I think it's nice to be able to give the option of more granular prices. Our requirement is an hour, but if you're generating a weekly report for your node, it's reasonable to want lower.